wrappers/src/error_handling.cpp

```diff
@@ -90,6 +91,9 @@ namespace realm {
         catch (const RowDetachedException& e) {
             return { RealmErrorType::RealmRowDetached, e.what() };
         }
+        catch (const MissingPrimaryKeyException& e) {
+            return { RealmErrorType::RealmTableHasNoPrimaryKey, e.what() };
```

Comment:
This isn't specific to this PR, but I think it's unfortunate that we are operating with three different naming schemes for these errors. Maybe we could rename our `RealmErrorType` entries to match core exceptions at some point..